src/command.cpp

```diff
@@ -506,10 +510,11 @@ std::string Command::tagged_cmd(const std::string & cmd, const uint8_t flag) {
 
 // list all commands for a specific device, output as json
 bool Command::list(const uint8_t device_type, JsonObject output) {
-    // force add info and commands for those non-EMS devices
+    // check of it a 'commands' command
     if (device_type == EMSdevice::DeviceType::TEMPERATURESENSOR || device_type == EMSdevice::DeviceType::ANALOGSENSOR) {
         output[F_(info)]     = Helpers::translated_word(FL_(info_cmd));
         output[F_(commands)] = Helpers::translated_word(FL_(commands_cmd));
+        output[F_(values)]   = Helpers::translated_word(FL_(values_cmd));
```

Comment:
For ems devices the values command is hidden. Please show it for all devices or for none, but don't mix.

Reply:
I saw that too and was thinking of making `values` not HIDDEN. It's the only command that is. What do you think?

Reply:
Ok for me, show it everywhere.

Reply:
ok. done.